roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -21,7 +21,7 @@
 
 - name: Stop if unknown OS
   assert:
-    that: ansible_distribution in ['RedHat', 'CentOS', 'Fedora', 'Ubuntu', 'Debian', 'CoreOS', 'Container Linux by CoreOS', 'openSUSE Leap', 'openSUSE Tumbleweed']
+    that: ansible_distribution in ['RedHat', 'CentOS', 'Fedora', 'Ubuntu', 'Debian', 'CoreOS', 'Container Linux by CoreOS', 'openSUSE Leap', 'openSUSE Tumbleweed', 'ClearLinux']
```

Comment:
could you change this to `ansible_os_family` too

Reply:
i will try this out in the next patch, though i do not have access to all the other OSes to verify that this change will work.

Reply:
CI will take care of that. :) We should be consistent about either using `ansible_distribution` or `ansible_os_family`